charts/policy-controller/README.md

```diff
@@ -103,12 +100,24 @@ cosign generate-key-pair
 
 This command generates two key files `cosign.key` and `cosign.pub`. Next, create a secret to validate the signatures:
 
+```shell
+kubectl create secret generic mysecret -n \
+
+As supported in previous versions, you could create your own key pair:
+
+```shell
+export COSIGN_PASSWORD=<my_cosign_password>
```

Comment:
this is already explained in https://github.com/sigstore/helm-charts/pull/348/files#diff-ac521e496f0c99485176eba85f33303525ed7043860fd6e510bbbf788ef395d2R98